File: Park20Management_App/src/app/_dtos/ParkWithDistance.dto.ts
import { ParkAddressDTO } from "./ParkAddress.dto";

export interface ParkWithDistanceDTO {
    designation: string,
    address : ParkAddressDTO;
    distance : number;
    spotHandicap : number;
    spotEletric : number;
    spotGPL : number;
    spotGas : number;
    spotMotorcycle: number;
  }

File: Park_Microservice/src/main/java/park20/Park_Microservice/dto/ParkWithDistanceDTO.java
package park20.Park_Microservice.dto;

import java.io.Serializable;

public class ParkWithDistanceDTO implements Serializable {
        public String id;
        public String designation;

        public ParkAddressDTO address;

        public double distance;

        public Integer spotHandicap;

        public Integer spotEletric;

        public Integer spotGPL;

        public Integer spotGas;

        public Integer spotMotorcycle;

}
